thermo_nerf/scripts/train_eval_script.py

```diff
@@ -63,6 +67,18 @@ def __post_init__(self) -> None:
 if __name__ == "__main__":
     parameters = tyro.cli(TrainingParameters)
 
+    if parameters.model_type == ModelType.NERFACTO:
+        tmp_folder = Path("./data_set")
+        shutil.copytree(src=parameters.data, dst=tmp_folder)
+        os.chmod(tmp_folder / "transforms.json", stat.S_IRWXU)
```

Comment:
Why is this needed?

Reply:
The copytree command will copy the permission status, therefore without changing the permission, the json file is read only.